app.js

```diff
@@ -1,14 +1,21 @@
 const express = require("express");
 const app = express();
 
-const { getTopics, getEndpoints } = require("./controllers/topics.controllers.js")
+const { getTopics, getEndpoints, getArticleById } = require("./controllers/article.controllers.js")
+
+app.use(express.json());
```

Comment:
Is this middleware needed at this stage where we only have a GET request? When do we need this middleware? We should only introduce new middleware, packages etc as and when we need them.